CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Change Log
 ==========
 
+Bugfix Issue #2 *(2016-01-14)*
+----------------------------
+
+Fixes #2
+Minimum scale never properly set when calling com.lyft.android.scissors.TouchManager.resetFor(...)
+Details:
+- setMinimumScale(...) in resetFor() prematurely called before this.bitmapWidth/Height were set.
+- computeLimit(...) in resetFor() called without proper scaling.
+- setMinimumScale(...) method was erroneously always set to 1.0f, when bitmap was smaller than viewport.
```

Comment:
This was by design as if the image is smaller than viewport, we want to leave it as is.